packages/t/tailscale/monitoring.yml

```
@@ -0,0 +1,6 @@
+releases:
+  id: 141585
+  rss: https://github.com/tailscale/tailscale/releases.atom
+# No known CPE, checked 2024-11-13
+security:
+  cpe: ~
```

Thanks for adding the monitoring file. Taillscale does have a CPE. For tailscale the file should be changed to
```suggestion
  cpe:
    - vendor: tailscale
      product: tailscale
```